lib/graphiti/query.rb

```diff
@@ -90,6 +90,9 @@ def sideloads
       @sideloads ||= begin
         {}.tap do |hash|
           include_hash.each_pair do |key, sub_hash|
+            if key.to_s.starts_with?('on__')
```

Comment:
We are parsing this key out in both the query and the serializer. Maybe extract a small `Graphiti::Util::PolymorphicIncludeDirective` class which does the string splitting and gives you the parts?

Reply:
I think this is a good comment, but I'm going to hold off on it for now to avoid the indirection. Maybe if we touch it again or add fields support.